src/notifications/screens/notifications.screen.js

```diff
@@ -493,22 +481,17 @@ class Notifications extends Component {
                 renderItem={this.renderItem}
                 ListEmptyComponent={
                   !isLoadingNewNotifications && (
-                    <View
-                      style={[
-                        styles.textContainer,
-                        { height: contentBlockHeight },
-                      ]}
-                    >
-                      <Text style={styles.noneTitle}>
+                    <TextContainer height={contentBlockHeight}>
```

Comment:
Shouldn't `TextContainer` styled definition read and apply the `height` prop?

Reply:
I thought that too but I found that you actually don't need to... It seems that styled.Views can apply values (including height) passed in as props directly and handles null values better. I built out a small example here: https://snack.expo.io/Bys9FZumz

I can change it if it's more explicit but I thought leaving it like this is a bit neater.

Reply:
> styled-components pass on all their props.

From [documents](https://www.styled-components.com/docs/basics#passed-props), I learned something new. :)